ape_ens/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .namehash import manual_namehash, namehash
+
+__all__ = ["namehash", "manual_namehash"]
```

Comment:
**todo**: let's not expose `manual_namehash` as a public API but rather keep it internal.

To do this, rename the method and it's usage to `_manual_namehash`.